dietpi/dietpi-software

```diff
@@ -12091,6 +12098,73 @@ If no WireGuard (auto)start is included, but you require it, please do the follo
 
 			unset -v choice_required domain port invalid_text server_ip
 		fi
+
+		if To_Install 213 # soju
+		then
+			local ver=0.8.0
+			aDEPS=('gcc' 'make' 'pkg-config' 'scdoc' 'libpam-dev' 'libsqlite3-dev')
+			Download_Install "https://git.sr.ht/~emersion/soju/archive/v${ver}.tar.gz"
```

Comment:
Let's get the latest version and archive from GitHub, so we do not need to regularly check and update it: https://api.github.com/repos/emersion/soju/releases/latest

See how we do that with the GitHub API in many other cases. We do define a fallback URL though, just in case asset names or structure change. But of not, users will always get the latest version.